src/plugins/str.rs

```diff
@@ -33,6 +35,26 @@ impl Str {
         let applied = match self.action.as_ref() {
             Some(Action::Downcase) => Value::string(input.to_ascii_lowercase()),
             Some(Action::Upcase) => Value::string(input.to_ascii_uppercase()),
+            Some(Action::Substring(s)) => {
+                // Index operator isn't perfect: https://users.rust-lang.org/t/how-to-get-a-substring-of-a-string/1351
+                let no_spaces: String = s.chars().filter(|c| !c.is_whitespace()).collect();
```

Comment:
You could also split, and then run .trim() on the results of the split

Reply:
Way more clean, especially in the new split-focus approach